### DISCUSSION TITLE
Accessing a port-forward url via a third party tool like Postman

### DISCUSSION THREAD

**Discussion by indoben** [coder/code-server#6341] [Q&A] [answered]
If I have a port forwarded from my workspace that works in the browser: (e.g. https://3000--main--admin-node--bwatanabe.mydomain.com). How do I authenticate to that endpoint using another tool like postman?

**Comment by code-asher** [comment, marked as answer]
It looks like this might be for coder/coder rather than 
coder/code-server?

But regardless, I think it should work if you send a token in a 
cookie.

https://learning.postman.com/docs/sending-requests/cookies/

It looks like it can even automatically grab the cookies if you 
use their Interceptor browser extension.

**Reply by code-asher** [reply]
Actually since the cookies are HTTP-only I am not sure the browser extension can get them...you might have to copy them manually.

**Reply by code-asher** [reply]
Ah nevermind I stand corrected, apparently the cookies for forwarded ports are not HTTP-only.  The main session token is though.

**Reply by indoben** [reply]
That worked, thank you!
